src/native/external/libunwind/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(libunwind)
 
-cmake_minimum_required(VERSION 3.16.1)
```

Comment:
We don't want to touch the external sources. These are projects that we've included the sources from as-is with as few changes as possible and that we include in our build scripts at various points. We shouldn't make changes in src/native/external if not required.

Reply:
Okok! I'll revert those in the next commit. I'm missing `tests`, `mono`, and `tasks`. Do we want to update those 3 subtrees?

Reply:
We want to update `tests` and `mono`. We don't want to update `tasks` as we ship those to customers (for building mobile apps) and we don't want to up that minimum.